distribution/docs/src/main/resources/_contents/_platform-contents/managing-platform-contents.adoc

```diff
@@ -58,3 +58,16 @@ The Platform application is installed by default on a standard installation.
 |
 
 |===
+
+==== Country Code Converter
+
+In the Platform Application, you have the option to install a feature called `platform-country-local`.
+This feature provides an OSGi service to convert country codes into various formats by performing a local lookup.
+
+In order to utilize the FIPS 10-4 to ISO 3166-1 alpha 3 and vice versa country code conversions, the `FIPS 10-4 to ISO 3166-1 alpha 3 File Location` field in the feature's configuration must be entered with the file path of a properties file containing the FIPS 10-4 to ISO 3166-3 alpha 3 country code mappings. However, the feature is already configured by default upon installation, with the file location field pointing to a properties file containing country code mappings from http://www.opengeocode.org/. This file can be found at: ${branding-lowercase}.home/etc/fipsToIso.properties, and should only be modified in the case the country code mappings would ever change.
```

Comment:
Don't think we typically use `${branding-lowercase}.home` to refer to the home directory. @ricklarsen?